app/src/androidTest/assets/features/create_rhythm.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Create a rhythm
+When you create a rhythm
+
+  Scenario Outline: Tapping too few times
+    Given I am viewing the create rhythm activity
+    When I tap the start button
+    And I tap the screen <tap_count> times
+    And I press the stop button
+    Then I get an error that I tapped not enough times
+
+    Examples:
+      | tap_count |
+      | 0         |
+      | 1         |
+      | 2         |
+
+  Scenario: Tapping enough times
```

Comment:
enough is not very well defined i think, you now use 10 but in the strings.xml you say at least 3 times